Packs/BmcITSM/Integrations/BmcITSM/BmcITSM_description.md

```diff
@@ -0,0 +1,9 @@
+## Access to BMC Helix ITSM
+- The credentials (username and password parameters) are used to authenticate with the ITSM API for the provided instance URL. 
+
+- The role of the user should have access to service requests, change requests, problem investigation, known error, task and incidents. Keep in mind that in order to delete a ticket the user must have an Admin role. 
```

Comment:
The user role should have access to service request, change request, problem investigation, known error, task, and incident tickets.
**Note:**
Only the Admin role can delete a ticket.